x-pack/test/functional/apps/ml/short_tests/model_management/model_list.ts

```diff
@@ -117,9 +128,10 @@ export default function ({ getService }: FtrProviderContext) {
         modelWithPipelineAndDestIndexExpectedValues.dataViewTitle
       );
       // Delete pipelines from deploy DFA model tests
-      await ml.api.deleteIngestPipeline(modelWithoutPipelineDataExpectedValues.name, false);
+      await ml.api.deleteIngestPipeline(modelWithoutPipelineDataExpectedValues.name, false, false);
       await ml.api.deleteIngestPipeline(
         modelWithoutPipelineDataExpectedValues.duplicateName,
+        false,
```

Comment:
what is the reason for skipping the assertion here?

Reply:
When there is a test failure, this also fails.  Felt like a false negative.  Again, I can remove and re-test on the morrow.  Less code is the best code? lol

Reply:
I've left this alone for now, but whatever you and your team prefer, will be done.

Reply:
@darnautov So what do you think on this bit now Dima?

Reply:
I'll take the silence as agreement for now :)

Reply:
I hope [this](https://github.com/elastic/kibana/pull/174045/commits/64361f9ac59270ce82f4673794938a87be98d603) is what is desired gentlemen.